src/handlers/handleNextQuestion.js

```diff
@@ -1,15 +1,20 @@
 'use strict';
-import creatSubmitBtn from '../views/creatSubmitElement.js';
+
 import showCurrentQuestion from "./showCurrentQuestion.js";
+import {NEXT_QUESTION_BUTTON_ID } from '../constants.js';
 import { quizData } from '../data.js';
+import getDOMElement from '../utils/getDOMElement.js';
 
 const handleNextQuestion = () => {
     quizData.currentQuestionIndex = quizData.currentQuestionIndex + 1;
-    if (quizData.currentQuestionIndex === quizData.questions.length-1) {
-        creatSubmitBtn();
+    
+    if (quizData.currentQuestionIndex === quizData.questions.length - 1) {
+        const nextQuestionButton = getDOMElement(NEXT_QUESTION_BUTTON_ID);
+        nextQuestionButton.innerText = 'Submit';
     }
-
+    if (quizData.currentQuestionIndex > 9) {
```

Comment:
Why nine?

Reply:
to block it from going further ... if it is 9 we will be in the last question ... if > 9 it will make next question for null . so this statement will prevent it from going further

Reply:
You should use `quizData.questions.length` instead, so you can add more questions without problems.